### PR TITLE
[DO NOT MERGE] Rebased BR translation

### PR DESCRIPTION
**Pull request by sagirk**
@marcelosdm I tried to resolve the merge conflicts on my own. Can you quickly check if this looks good (and that I didn't lose any of your changes during the rebase)?

If all is good, we can close this PR and then I will directly push these changes to the translation branch, which we can then merge into master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,13 +279,15 @@ function someFunction()
...
   // do something amazing
-}());
+})();
 
 // Avoid — throws exception
-const count = 2 // it tries to run 2(), but 2 is not a function
-(function doSomething() {
-  // do something amazing
-}())
+const count = 2(
```
Comment (BrunoScheufler) [inline README.md]: Something broke here, guess it's due to the missing semicolon which shifted it up

**Comment by marcelosdm** [comment]
> @marcelosdm I tried to resolve the merge conflicts on my own. Can you quickly check if this looks good (and that I didn't lose any of your changes during the rebase)?
> 
> If all is good, we can close this PR and then I will directly push these changes to the translation branch, which we can then merge into master.

Hello guys!
@sagirk well, I think it's ok and you're not missing anything!

Thanks so much!

File: README.md (hunk 1 above)
Comment (sagirk) [reply README.md]: Good catch, @BrunoScheufler! 👍 

I think prettier auto-formatted the code blocks following my local config.

Let me comb through the entire file again to see if there are more such instances.

**Comment by sagirk** [comment]
Fixed auto-formatting issues. Looks good now. 

Closing this and opening a proper PR to merge it into master.

**Pull request closed without merging**